internal/config/config_test.go

```diff
@@ -0,0 +1,260 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+package config
+
+import (
+	"bytes"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestRead(t *testing.T) {
+	require := require.New(t)
```

Comment:
I think we don't need this line. Could you check?

Reply:
This allows us to call the require functions, like `require.Equal`, without having to pass the `t *testing.T` parameter all the time.